### ISSUE TITLE
Is it okay to run the larbs.sh script more than once on a system

### ISSUE THREAD

**Issue by khang33k** [LukeSmithxyz/LARBS#232]
So, i ran this script once and the config files were not downloaded and startx wouldn't work so I ran the script again and the configs got downloaded and it worked.

This is not an issue but just wondering if it is safe to run this script twice on a system?
As it changes /etc/makepkg.conf and enables multicores

so i am just a bit worried if running it again would cause some terrible changes in the makepkg.conf?
As I don't know bash or sed etc which is used in the script, that's why. 
In short my question is will it be fine to run this script as root more than once on a system?

**Comment by LukeSmithxyz** [comment]
Yeah it's safe. The sed command isn't a problem, although if you manually revert one of the changes, running the script will obviously change it. It won't mess the file up if run multiple times though.
